DEVELOPERS.md

```diff
@@ -171,58 +109,52 @@ than with Ruby.
 
 Node.js is required to work with frontend tooling such as tailwind-cli.
 
-Install Node.js. `asdf` will consult the `.tool-versions`
+Install Node.js. `mise` will consult the `.tool-versions`
 file to select the version.
 
 ```sh
-$ asdf install nodejs
+$ mise install nodejs
```

Comment:
Since we have `nodejs` and `postgres` in our `.tool-versions` file, do we need to run `mise install nodejs` separately? Doesn't `mise install` install everything?

Reply:
it does, I was going to take another pass to try to make it more idiomatic after getting rid of asdf. Unfortunately the "do it all" does stuff like expose you to _every_ system dependency requirement/compiler/etc.

Reply:
Another thing I need to add in is how to hook in into the shell